src/styles/blocks/nav.scss

```
@@ -0,0 +1,41 @@
+.nav {
+  &__list {
+    display: flex;
+  }
+
+  &__item {
+    &:not(:last-child) {
+      margin-right: 20px;
+    }
+  }
+
+  &__link {
+    padding: 23px 0;
+    text-decoration: none;
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 60px;
+    text-transform: uppercase;
+    white-space: nowrap;
+    color: #000;
```

Nav links color is not black any more
```suggestion
    color: $color-regular;
```